internal/servers/controller/controller.go

```diff
@@ -140,9 +141,11 @@ func New(ctx context.Context, conf *Config) (*Controller, error) {
 			client, cleanup, err := external_host_plugins.CreateHostPlugin(
 				ctx,
 				pluginType,
-				external_host_plugins.WithHostPluginsFilesystem("boundary-plugin-host-", host_plugin_assets.FileSystem()),
-				external_host_plugins.WithHostPluginExecutionDir(conf.RawConfig.Plugins.ExecutionDir),
-				external_host_plugins.WithLogger(hclog.NewNullLogger()))
+				external_host_plugins.WithPluginOptions(
+					pluginutil.WithPluginExecutionDirectory(conf.RawConfig.Plugins.ExecutionDir),
+					pluginutil.WithPluginsFilesystem("boundary-plugin-host-", host_plugin_assets.FileSystem()),
```

Comment:
perhaps a const since it's used in a few places?

Reply:
Done